music21/lily/translate.py

```diff
@@ -2302,16 +2302,21 @@ def setHeaderFromMetadata(self, metadataObject=None, lpHeader=None):
         lpHeaderBodyAssignments = lpHeaderBody.assignments
 
         if metadataObject is not None:
-            if metadataObject.title is not None:
+            titles = metadataObject['title']
+            if titles:
                 lyTitleAssignment = lyo.LyAssignment(assignmentId='title',
                                                      identifierInit=lyo.LyIdentifierInit(
-                                                         string=metadataObject.title))
+                                                         string=str(titles[0])))
```

Comment:
Yes, and it would be easier if I hadn't done the metadataObject.title returns 'MULTIPLE' thing, which is what I'm working around here.  Damn, I really liked returning 'MULTIPLE', but maybe returning the first item is better?  My API instincts are saying "give the client the choice", but that gets weird and horrible and hard to describe:

`md.setPreference('returnFirstElementFromAttributesIfMultipleElementsExist', True) # defaults to False (return 'MULTIPLE')`

Reply:
Waiting for resolution of what we return instead of 'MULTIPLE'.

Reply:
Done.